mmv1/templates/terraform/pre_delete/alloydb_instance.go.erb

```diff
@@ -0,0 +1,18 @@
+// Read the config and avoid calling the delete API if the instance_type is SECONDARY and instead return nil
```

Comment:
Can you update the documentation of the resource to state that this will happen for SECONDARY instances? Two of the resource description, instanceType's description, and a `docs` block (https://github.com/GoogleCloudPlatform/magic-modules/blob/85d3f6e001151c63845fd72325ab07e55f558429/mmv1/api/resource.rb#L158) should be enough

Reply:
@melinath we don't document `docs` blocks, right?

Reply:
the docs block is currently documented on https://googlecloudplatform.github.io/magic-modules/develop/resource/

Reply:
Oh, neat! I missed it.

Reply:
Added documentation in resource docs block and instanceType description.